### PR TITLE
Fix sdl2-config behavior by restoring libSDL2.so

### PR DESCRIPTION
**Pull request by elizagamedev**
There is a bug in SDL2's CMakeLists.txt which tries to copy libSDL2.so
from an incorrect location. This behavior had previously been patched
out, which had the unfortunate side effect of breaking the behavior of
sdl2-config. This behavior has been corrected.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by uilianries** [comment]
Hi @elizagamedev !

Thanks for reporting and fixing this! I'll wait for CI result before to merge.

Regards!

**Comment by elizagamedev** [comment]
Hmm, looks like the symlink was screwy. I think it might need to be manually recreated. I'll work more on it tomorrow.

**Comment by elizagamedev** [comment]
Okay, it's not the most ideal, but I think I've fixed the problem.

**Comment by elizagamedev** [comment]
Ack. Okay, this bug was dumb, but it looks like it's been fully fixed at this point.

**Pull request merged**
